libparsec/crates/types/src/invite.rs

```diff
@@ -65,7 +64,6 @@ impl ToString for InvitationType {
 
 // SAS code is composed of 4 hexadecimal characters
```

Comment:
The comment seems out of place now, I think it should be move to a doc-comment of the struct `SASCode`

Reply:
I moved the comment to make it a doc-comment